### ISSUE TITLE
Unable to use filters reliably

### ISSUE THREAD

**Issue by adamgibbins** [rawdigits/wee-slack#283]
I want to filter bots from certain channels. On IRC I'd do something similar to `/filter add somechannel_github irc.freenode.#somechannel nick_GitHub *` to filter the GitHub nick from #somechannel. However in Slack channels this does not work.

If I toggle `/debug tags` weechat shows the tags for every message, it looks similar to this for an IRC message: `irc_privmsg,notify_message,prefix_nick_85,nick_GitHub,host_~GitHub@somehost,log1`

However on Slack it looks like this:
`notify_message,log1,irc_privmsg` - i.e. all the nick related filters are missing, which I think is why I'm unable to filter so I've no way of distinguishing bots without using some nasty string based regex.

**Comment by ToxicFrog** [comment]
`nick_*` tags are working. We don't set `irc_privmsg` tags, though -- it's an open question as to whether we *should*, when perhaps it should be `slack_message` or similar, but OTOH if we don't set `irc_privmsg` stuff like nick highlights don't work with the default weechat configuration.

My preference would be to set both `irc_privmsg` and `slack_message`, or similar.

**Comment by trygveaa** [comment]
@ToxicFrog: We do set `irc_privmsg` on dm messages ([see here](https://github.com/wee-slack/wee-slack/blob/3eff1de49d3aba1d991b7b6953e6b55a24fdecd9/wee_slack.py#L2848)).

I tested adding a filter now, and it works fine for me, so I think we can close this.

**Comment by ToxicFrog** [comment]
They should be set on channel messages as well, though, in keeping with the behaviour of weechat WRT IRC channels.

**Comment by trygveaa** [comment]
Ah, the name made me think they only were for dms, but yes, I see now that they should be on channel messages as well.

**Comment by ToxicFrog** [comment]
Yeah, confusingly, the irc PRIVMSG command is used for both PMs and channel messages, and weechat applies the tag to both.
